Examples.fshx

```diff
@@ -53,32 +133,154 @@ Instance: mCODEKarnofskyPerformanceStatusExample01
 * valueCodeableConcept = LNC#LA29177-5 "Normal activity with effort; some signs or symptoms of disease"
 
+
 Instance: mCODECancerRelatedMedicationStatementExample01
 InstanceOf: CancerRelatedMedicationStatement
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedMedicationStatement"
 * status = #active "active"
-* medicationCodeableConcept = RXN#93698 "Carboplatin Injectable Solution [Paraplatin]"
+* category = #community "community"
+* medicationCodeableConcept = RXN#349472 "gefitinib 250 MG Oral Tablet"
 * subject = Reference(mCODEPatientExample01)
 * effectiveDateTime = "2019-04-01"
 * dateAsserted = "2019-04-01"
-* extension.valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
 * extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
+* dosage.text = "250mg orally once daily with or without food"
+* dosage.route = SCT#26643006 "Oral use"
+* dosage.dose[x].system = UCUM#mg "mg"
+* dosage.dose[x].value = "250"
 
 
 Instance: mCODECancerRelatedSurgicalProcedureExample01
 InstanceOf: CancerRelatedSurgicalProcedure
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedSurgicalProcedure"
 * status = #completed "completed"
+* category = SCT#387713003 "Surgical procedure"
 * code = SCT#359615001 "Partial lobectomy of lung (procedure)"
 * subject = Reference(mCODEPatientExample01)
 * asserter = Reference(mCODEPractitionerExample01)
 * performedDateTime = "2019-03-01"
 * extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
 * reasonReference = Reference(mCODEPrimaryCancerConditionExample01)
 * bodySite = SCT#41224006 "Structure of lower lobe of left lung (body structure)"
+
+
+Instance: mCODECancerRelatedRadiationProcedureExample01
+InstanceOf: CancerRelatedRadiationProcedure 
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedRadiationProcedure"
+* status = #completed "completed" 
+* category = SCT#53438000 "Radiation therapy procedure or service (procedure)"
+* code = SCT#152198000 "Brachytherapy (procedure)"
+* subject = Reference(mCODEPatientExample01)
+* asserter = Reference(mCODEPractitionerExample01)
+* performedDateTime = "2019-03-01"
+* extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
+* extension[RadiationDose].extension[totalRadiationDoseDelivered].valueQuantity.code = UCUM#cGy 
+* extension[RadiationDose].extension[totalRadiationDoseDelivered].valueQuantity.value = "1200"
+* reasonReference = Reference(mCODEPrimaryCancerConditionExample01)
+* bodySite = SCT#41224006 "Structure of lower lobe of left lung (body structure)"
+
+
+Instance: mCODETNMClinicalDistantMetastasesCategoryExample01
+InstanceOf: TNMClinicalDistantMetastasesCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalDistantMetastasesCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
```

Comment:
This should already be set.

Reply:
Not in the profile.

Reply:
This might be the wrong category, anyway. Clinical staging comes from an examination, not a lab test. Is there an physical exam category?

Reply:
Just survey.  Already noted in the comment below... just change the clinical staging examples to * category = #survey "Survey"